services/api/sqlc.yml

```
@@ -17,3 +17,11 @@ sql:
         emit_json_tags: true
         emit_prepared_queries: true
         emit_interface: true
+        emit_pointers_for_null_types: true
+        overrides:
+        - db_type: "uuid"
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 8

(indentation)